app/test/unit/git/commit-test.ts

```diff
@@ -514,24 +516,75 @@ describe('git/commit', () => {
     })
   })
 
-  describe('createMergeCommit with a merge conflict', () => {
-    let repository: Repository
-    describe('with a merge conflict', () => {
+  describe('createMergeCommit', () => {
+    describe('with a simple merge conflict', () => {
+      let repository: Repository
+      describe('with a merge conflict', () => {
```

Comment:
:art: does this seem necessary given another `describe` has been inserted two lines above?

Reply:
I ask because it looks like the outer case seems to be enough to describe the test case:

```
 createMergeCommit
  - with a simple merge conflict
     - creates a merge commit
```

with this additional `describe`:

```
 createMergeCommit
  - with a simple merge conflict
     - with a merge conflict
        - creates a merge commit
```